src/singularity.ts

```diff
@@ -349,6 +349,7 @@ replication.command('start')
   .option('-u, --url-prefix <urlprefix>', 'URL prefix for car downloading. Must be reachable by provider\'s boostd node.', 'http://127.0.0.1/')
   .option('-p, --price <maxprice>', 'Maximum price per epoch per GiB in Fil.', '0.000000002')
   .option('-r, --verified <verified>', 'Whether to propose deal as verified. true|false.', 'true')
+  .option('-s, --start-delay <startdelay>', 'Deal start delay in days. (StartEpoch)', '7')
```

Comment:
I'd recommend https://www.npmjs.com/package/parse-duration
Because you never know what werid delay a client would like to provide, i.e. 100 hours.

Same for duration

Reply:
I think leave this for future improvement.